lib/platform/azure/__snapshots__/index.spec.ts.snap

```diff
@@ -188,6 +188,26 @@ Array [
 ]
 `;
 
+exports[`platform/azure updatePr(prNo, title, body) should reopen the PR 1`] = `
+Array [
+  Array [
+    Object {
+      "status": 1,
+    },
+    "1",
+    1234,
+  ],
+  Array [
+    Object {
+      "description": undefined,
```

Comment:
Description is undefined for tests, but PR body is present in [real-world example](https://dev.azure.com/szharinov/renovate-reopen-x/_git/renovate-reopen-x/pullrequest/6)

Reply:
@zharinov That's funky... But it also works in a live test for me. Are you happy to merge like this?

Reply:
We're already having this for other azure tests too, so yes. I'll take a look at `description: undefined` issue separately.